editor/src/components/canvas/ui-jsx-canvas-errors.spec.tsx

```diff
@@ -32,45 +32,50 @@ describe('UiJsxCanvas errors', () => {
     expect(canvasErrors).toMatchInlineSnapshot(`
       Array [
         Object {
-          "columnNumber": 32,
-          "lineNumber": 5,
           "message": "MyCard is not defined",
           "name": "ReferenceError",
-          "originalCode": Array [
-            ScriptLine {
-              "content": "    import * as React from \\"react\\"",
-              "highlight": false,
-              "lineNumber": 2,
-            },
-            ScriptLine {
-              "content": "    import { View, Storyboard, Scene } from 'utopia-api'",
-              "highlight": false,
-              "lineNumber": 3,
-            },
-            ScriptLine {
-              "content": "",
-              "highlight": false,
-              "lineNumber": 4,
-            },
-            ScriptLine {
-              "content": "    export var App = props => <MyCard data-uid={'bbb'} />",
-              "highlight": true,
+          "stackFrames": Array [
+            Object {
+              "columnNumber": 32,
+              "fileName": "code.tsx",
```

Comment:
Is this filename relative to the file the exception originates from?

Reply:
It's the actual file name. That's just what we're using in the test

Reply:
I was only asking because some of the filenames in the snapshots were prefixed by a `/` and most are not, like this one has a mix in the same set of stack frames: https://github.com/concrete-utopia/utopia/pull/1429/files/7d5aebd3b147fb1eaaa75a9098a026d3f3afc1d7#diff-16cbaeb060f62d1bad49e833b2e2eb5a955edba01a78dc60e7f189a2bf63082dR1035

Reply:
I'm pretty sure this is just an artefact of the test. The actual result on a real project shows the file paths from the root of the project (see the image above)